packages/localizer-date-fns/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "clean": "rimraf dist/* lib/*",
     "dist": "node ../../tools/build-localizer.js",
-    "build": "babel localizer.js -o index.js",
+    "build": "babel --config-file ../../babel.config.js localizer.js -o index.js",
```

Comment:
i don't think you need to explicitly provide the config if using `babel.config.js`

Reply:
basically, you have to, because lerna runs build scripts inside an each directory, so babel doesn't know correct root dire of the package
for this particular line we can remove `--config-file` parameter, but only because in localizer.js we don't use any features which has to be transpiled by babel (you can check this by adding spread operator in localizer.js). I personally would prefer keeping this parameter to not break build after someone decided to add expiremental ECMA features in these scripts

Reply:
btw, here is the original answer from Babel maintainer https://babeljs.slack.com/archives/C0DFJT81H/p1527961301000007

Reply:
Ah ok, If that's the case then i'd prefer to use individual `babelrc.s` that use the new `extends` keyword rather then the config option, since i sometimes run babel outside the one npm script.

Reply:
actually, I was stuck with the second method with updating `tools/build-localizer.js` webpack config to make it work with individual babelrc files inside each package. But I can make another trym if you want